@here/harp-mapview/lib/MapView.ts

```diff
@@ -739,8 +748,10 @@ export class MapView extends THREE.EventDispatcher {
             0.1,
             4000000
         );
+        this.camera.up.set(0, 0, 1);
         this.m_lookAtDistance = 0;
         this.m_focalLength = 0;
+        this.m_scene.add(this.m_camera); // ensure the camera is added to the scene.
```

Comment:
shouldn't this be moved into setupCamera?

Reply:
I'm not sure, but I don't have strong feelings about it. My point is: both the camera and the scene are created in the body of the constructor but then later in some other function (`setupCamera()`) the camera is added to the scene. I find this confusing.

Reply:
fair point